DEVELOPING_OPAL/demos/src/main/scala/org/opalj/ai/CipherGetInstanceStrings.scala

```diff
@@ -62,14 +56,14 @@ object CipherGetInstanceStrings extends ProjectsAnalysisApplication {
             val result = BaseAI(m, new DefaultDomainWithCFGAndDefUse(project, m))
             val code = result.domain.code
             for {
-                PCAndInstruction(pc, INVOKESTATIC(Cipher, false, "getInstance", _)) <- code
+                PCAndInstruction(pc, INVOKESTATIC(ClassType.Cipher, false, "getInstance", _)) <- code
```

Comment:
These seem very specific. At least consider prefixing the class name with the package?